src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/RedundantImportCheck.java

```diff
@@ -48,6 +48,16 @@
  * &lt;module name="RedundantImport"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package Test; // OK
+ * import Test.test; // violation, Imported from the same package as the current package
+ * import java.lang.String; // violation, the class non-statically imported is from the package
+ * import java.lang.String; // violation, It is a duplicate of another import
+ * public class test{ }; // OK
```

Comment:
This is not an import, we do not need `// OK` comment here, and please rename to `MyClass`or something like this to follow naming camelcase convention.

Reply:
Resolved.